util/feature.txt

```diff
@@ -0,0 +1,3 @@
+feature 1 implementation
+feature 2 implementation
+feature 3 implementation
```

Comment:
Let's add a bit more to the feature 2 implementation. I think it should handle multiple widgits.